lab-daniel/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
TODO: The extension name of this needs to be `.json` for your linter to actually work.